### PR TITLE
APR-1841 - management hierarchy content

### PR DESCRIPTION
**Pull request by gdbarnes**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml [hunk 1]
```diff
@@ -23,12 +26,12 @@
     <label for="FullName" class="govuk-label">Full name</label>
     @if (showErrors)
     {
-        <span class="govuk-error-message">
             @foreach(var nameErrorMessage in errorMessages.Where(x => x.Field == "FullName"))
             {
-                @Html.Raw(nameErrorMessage.ErrorMessage) <br />
+                <span class="govuk-error-message">
+                    @Html.Raw(nameErrorMessage.ErrorMessage)
```
Comment (martin-hornsby) [inline src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml:32]: Not sure why this is being output as @Html.Raw, can you remove that bit?

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml [hunk 2]
```diff
@@ -37,12 +40,12 @@
     <label for="JobRole" class="govuk-label">Job role</label>
     @if (showErrors)
     {
-        <span class="govuk-error-message">
             @foreach(var jobRoleErrorMessage in errorMessages.Where(x => x.Field == "JobRole"))
             {
-                @Html.Raw(jobRoleErrorMessage.ErrorMessage)  <br />
+                <span class="govuk-error-message">
+                    @Html.Raw(jobRoleErrorMessage.ErrorMessage)
```
Comment (martin-hornsby) [inline src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml:46]: Not sure why this is being output as @Html.Raw, can you remove that bit?

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml [hunk 3]
```diff
@@ -54,14 +57,14 @@
     </span>
     @if (showErrors)
     {
-        <span class="govuk-error-message">
             @foreach (var timeInRoleMonthsMessage in errorMessages.Where(x => x.Field == "TimeInRole"))
             {
-                @Html.Raw(timeInRoleMonthsMessage.ErrorMessage) <br />
+                <span class="govuk-error-message">
+                    @Html.Raw(timeInRoleMonthsMessage.ErrorMessage)
```
Comment (martin-hornsby) [inline src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml:63]: Not sure why this is being output as @Html.Raw, can you remove that bit?

**Review by martin-hornsby — DISMISSED** [review]

**Review by martin-hornsby — DISMISSED** [review]

File: src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml (hunk 1 above)
Comment (gdbarnes) [reply src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml]: I've removed all the @Html.Raw()'s throughout these views.

**Pull request merged**